setup.cfg

```diff
@@ -10,3 +10,27 @@ commit_parser = semantic_release.history.tag_parser
 minor_tag = [minor]
 fix_tag = [fix]
 branch = main
+
+[flake8]
+ignore = E226, E231, E501
+
+[coverage:report]
```

Comment:
Could you add a coverage report to CircleCI?

Reply:
Hmmm :thinking: yeah, I could look into that!

Reply:
related: https://github.com/WIPACrepo/wipac-dev-tools/pull/7#discussion_r632633474